conda_build/index.py

```diff
@@ -37,11 +38,16 @@
 import libarchive
 import conda_package_handling.api
 
+#  BAD BAD BAD - conda internals
+from conda.core.subdir_data import SubdirData
+from conda.models.channel import Channel
```

Comment:
Shouldn't these also come via `conda_build.conda_interface`?

Reply:
Definitely, but they're not current exposed in conda's exports.  It's really bad, and this needs a better way to do things without reaching into conda's internals.  I'm not sure it should block this PR and 3.18.0, though.